src/Paprika/Store/FanOutPage.cs

```diff
@@ -139,4 +139,7 @@ public void Accept(IPageVisitor visitor, IPageResolver resolver, DbAddress addr)
             }
         }
     }
+    public Page Destroy(IBatchContext batch, in NibblePath prefix){
+        return new Page();
```

Comment:
Missing impl.

Reply:
For these methods, I have added as a placeholder since it was throwing build error. These classes are implementing PagedB, so had to implement Destroy. 

For implementing, do I need to follow the same step as SStorageFanOutPage?